tools/cla_check.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# XXX copied from https://github.com/kyrofa/cla-check
+import collections
+import os
+import sys
+from subprocess import check_call, check_output
+
+
+try:
+    from launchpadlib.launchpad import Launchpad
+except ImportError:
+    sys.exit('Install launchpadlib: sudo apt install python-launchpadlib')
+
+Commit = collections.namedtuple('Commit', ['email', 'hash'])
+
+
+def get_commits_for_range(range):
+    output = check_output(['git', 'log', range, '--pretty=%aE|%H'])
+    split_output = (i.split('|') for i in output.split('\n') if i != '')
+    commits = [Commit(email=i[0], hash=i[1]) for i in split_output]
+    return commits
+
+
+def get_emails_from_commits(commits):
+    emails = set()
+    for c in commits:
+        output = check_output(['git', 'branch', '--contains', c.hash])
+        if '* master' not in output.split('\n'):
```

Comment:
This of course means that the user is _on_ master. Is that really a requirement?